### PR TITLE
Fix deploy branches config

### PR DESCRIPTION
**Pull request by mkleszcz**
Add lambda function to trigger and pass valid source reference to entrypoint.

### CODE REVIEW

File: infra/cdk/lib/stacks/env/ci/functions/trigger-entrypoint/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+module.exports.handler = async(event, context) => {
+    console.log('Start trigger entrypoint', { PROJECT_ENV_NAME: process.env.PROJECT_ENV_NAME, event });
+
+    try {
+        const codebuild = new AWS.CodeBuild();
+        const branches = JSON.parse(process.env.DEPLOY_BRANCHES || '') || [];
+        const projectName = process.env.PROJECT_NAME;
+        if (!projectName) {
+            const message = 'Invalid project name';
+            console.log(message);
```
Comment (pziemkowski) [inline infra/cdk/lib/stacks/env/ci/functions/trigger-entrypoint/index.js:15]: console.log to be removed

File: infra/cdk/lib/stacks/env/ci/functions/trigger-entrypoint/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                console.log('Codebuild started!', response);
+            }).
+            on('error', function(error, response) {
+                console.log('Run codebuild error', error, response);
+            }).promise();
+        }
+
+        return { event, branches, filteredBranches };
+    } catch (err) {
+        console.error(err);
```
Comment (pziemkowski) [inline infra/cdk/lib/stacks/env/ci/functions/trigger-entrypoint/index.js:42]: Is this expected? Wouldn't we want this lambda to fail in this scenario?

**Review by pziemkowski — CHANGES_REQUESTED** [review]
I added some smaller comments.

**Review by khanek — APPROVED** [review]

**Comment by khanek** [comment]
@pziemkowski Could you take a look at PR again?

**Review by pziemkowski — APPROVED** [review]

**Pull request merged**